merkletree/proof.go

```diff
@@ -88,13 +89,22 @@ func VerifyAuthPath(ap *AuthenticationPath,
 	return true
 }
 
+func VerifyCommitment(salt []byte, key string, value []byte, commitment []byte) bool {
+	got := crypto.Digest(salt, []byte(key), value)
```

Comment:
I'm thinking about this:

```
// package crypto
func Commitment(key string, value []byte) (salt []byte, commit []byte, err error) {
	salt, err = MakeRand()
	if err != nil {
		return nil, nil, err
	}
	return salt, Digest(salt, []byte(key), value), nil
}
```
and make `VerifyCommitment` be a function of `crypto` package.
What do you think?